client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts

```diff
@@ -17,6 +17,7 @@ export const A4A_MARKETPLACE_LINK = '/marketplace';
 export const A4A_MARKETPLACE_PRODUCTS_LINK = `${ A4A_MARKETPLACE_LINK }/products`;
 export const A4A_MARKETPLACE_HOSTING_LINK = `${ A4A_MARKETPLACE_LINK }/hosting`;
 export const A4A_MARKETPLACE_HOSTING_REFER_ENTERPRISE_LINK = `${ A4A_MARKETPLACE_HOSTING_LINK }/refer-enterprise-hosting`;
+export const A4A_MARKETPLACE_HOSTING_REFER_PREMIUM_PLAN_LINK = `${ A4A_MARKETPLACE_HOSTING_LINK }/refer-premium-plan`;
```

Comment:
Nitpicking: Would it be better to explicitly include the pressable in the name?

Reply:
Thanks for the review, @jkguidaven!

Updated it here: https://github.com/Automattic/wp-calypso/pull/105276/commits/7e1f9c8d9b73839dced900b1ec65fb6255bb3ec7